### PR TITLE
Progression tests v2

### PR DESCRIPTION
**Pull request by dav1312**
Refactor for easier future modifications and also set test branch and signature

### CODE REVIEW

**Comment by vondele** [comment]
I might tag SF 15.1 but we still keep testing against SF 15, so fully automatic based on the latest tag might not work.

Bonus points for an automatic PT test Info message that picks the commit header, commit date, and formulates something like `PT of "Remove an unneeded randomization of evals." of Aug 12th vs SF 15.` not necessarily for this PR.

File: server/fishtest/templates/tests_run.mak [hunk 1]
```diff
@@ -517,7 +527,7 @@
       if (btn.dataset.options) testOptions = btn.dataset.options;
 
       if (testOptions) {
-        const { tc, new_tc, threads, options, book, stop_rule, bounds, games, base_branch, base_signature } = JSON.parse(testOptions);
+        const { tc, new_tc, threads, options, book, stop_rule, bounds, games, test_branch, base_branch, test_signature, base_signature } = JSON.parse(testOptions);
```
Comment (ppigazzini) [inline server/fishtest/templates/tests_run.mak:530]: @dav1312 what are doing these new parameters?

Comment (dav1312) [reply server/fishtest/templates/tests_run.mak:530]: Set the test branch and the test signature...?

Comment (ppigazzini) [reply server/fishtest/templates/tests_run.mak:530]: I'm testing the PR on DEV but I don't know how to check these new parameters vs master. I need some help to understand the high level function they are used for (eg new test, reschedule, back button etc.) :)

Comment (dav1312) [reply server/fishtest/templates/tests_run.mak:530]: It's just for this
image

And if it is a reschedule pressing the PT presets won't change those values

Comment (ppigazzini) [reply server/fishtest/templates/tests_run.mak:530]: Thank you :)

**Review by ppigazzini — APPROVED** [review]

**Comment by ppigazzini** [comment]
PROD updated, thank you @dav1312 :)

**Pull request merged**